contrib/sarplus/README.md

```diff
@@ -0,0 +1,191 @@
+# sarplus (preview)
+pronounced sUrplus as it's simply better if not best!
+
+[![Build Status](https://dev.azure.com/best-practices/recommenders/_apis/build/status/contrib%20sarplus?branchName=master)](https://dev.azure.com/best-practices/recommenders/_build/latest?definitionId=107&branchName=master)

```

Comment:
Understanding that we are having a different pipeline for SAR+, do we want to consolidate the build status with other pipelines? 

Maybe also good to add it, and perhaps the other contrib in the future if there are more to come, to the [testing strategy](https://github.com/Microsoft/Recommenders/wiki/Test-Strategy) to make it clear.

Reply:
not sure what you mean. do you want to consolidate and build this as part of the other pipelines? Since sarplus doesn't have dependencies to other parts of the repo, building it independently would save time and would allow us to release binaries from there.

Reply:
Ah no I mean do we want to put the build status on the same README and show them on the front page of the repo.

Reply:
understood. good idea!

added a section to the end of the README under contribution builds